charts/sbombastic/values.yaml

```diff
@@ -83,6 +83,8 @@ worker:
     requests:
       cpu: 250m
       memory: 300Mi
+  trivyDBRepository: public.ecr.aws/aquasecurity/trivy-db
+  trivyJavaDBRepository: public.ecr.aws/aquasecurity/trivy-java-db
```

Comment:
Is there no need to specify the protocol (http/https)?

Reply:
Sorry, my fault. They have their own container registry. That's totally fine then. Ignore my comment.